libass/ass_types.h

```diff
@@ -22,6 +22,52 @@
 
 #include <stdint.h>
 
+/**
+ * GENERAL NOTE regarding the definitions exposed by this header
+ *
+ * The main use case for this is _reading_ the track fields, especially
+ * track->YCbCrMatrix, to correctly display the rendering results.
+ *
+ * Furthermore the exposed definitions also open up the possibility to _modify_
```

Comment:
Would be worth sprucing up the grammar here a bit: "Furthermore", "e.g.", "well-versed", "internals,", "API breaks"…